source/importers/SynchronizeColorsTokens.ts

```diff
@@ -0,0 +1,218 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = require('./configs/SynchronizeColorsTokensConfig.json');
+
+interface IResultFileJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+interface IResultJsonObject  extends IResultFileJsonObject {
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonColors: Array<any>
+) => {
+	let hexToCssVariable: Object = {};
+	try {
+		let result: Array<string> = [];
+		let temporaryColorsJson = {};
```

Comment:
`Dict<Array<IResultJsonObject>>`?
Similar for line below

Reply:
Thanks. I used it. I would like to make interfaces and function resuable, but not now. In next usage in DS in similar script.